src/nuclide.cpp

```diff
@@ -301,7 +301,7 @@ Nuclide::Nuclide(hid_t group, const vector<double>& temperature)
   }
 
   // Check for total nu data
-  if (object_exists(group, "total_nu")) {
+  if (object_exists(group, "total_nu") && settings::create_delayed_neutrons) {
```

Comment:
@cfichtlscherer I know we had talked about this offline. On second thought, I think the most rigorous way of handling this option is to check `create_delayed_neutrons` inside the `Nuclide::nu` method. When `EmissionMode::delayed` is passed, `nu(...)` should return 0 and when `EmissionMode::total` is passed, it should return the prompt value. This ensures that any other uses of `nu(...)` reflect the same option (e.g., in tallies).
```suggestion
  if (object_exists(group, "total_nu")) {
```